src/main/java/org/symphonyoss/symphony/messageml/elements/Button.java

```diff
@@ -43,11 +43,12 @@
 
   public static final String MESSAGEML_TAG = "button";
   public static final String ACTION_TYPE = "action";
+  public static final String CANCEL_TYPE = "cancel";
   public static final String RESET_TYPE = "reset";
 
   private static final Set<String> VALID_CLASSES = new HashSet<>(Arrays.asList("primary", "secondary", "tertiary", "destructive",
       "primary-destructive", "secondary-destructive")); // primary-destructive, secondary-destructive are deprecated
-  private static final Set<String> VALID_TYPES = new HashSet<>(Arrays.asList(ACTION_TYPE, RESET_TYPE));
+  private static final Set<String> VALID_TYPES = new HashSet<>(Arrays.asList(ACTION_TYPE, RESET_TYPE, CANCEL_TYPE));
```

Comment:
Should this new type be allowed only if the button is inside a Dialog or we can use it also for simple forms?

Reply:
It looks like it will only be used for dialogs but I guess putting it in a regular form won't break anything but just will be useless?

Reply:
According to Jean-Michael, we don't want to limit the new type to Dialog use cases, they could be used in other situations later